addons/survey/views/survey_question_views.xml

```diff
@@ -109,6 +114,7 @@
                                 <tree editable="bottom">
                                     <field name="sequence" widget="handle"/>
                                     <field name="value" string="Choices"/>
+                                    <field name="image" string="Image" options="{'accepted_file_extensions': 'image/*'}"  attrs="{'column_invisible': [('parent.answer_image', '=', False)]}"/>
```

Comment:
Does it not only accept images by default?

Also, no need to repeat `string="Image"` as it's already the field name.
You only set the string attribute when you want something different from the label on the field definition.

Reply:
By default, it accepts more than images, for example I can also select PDF.